documentcloud/documents.py

```diff
@@ -345,86 +345,219 @@ def _upload_file(self, file_, **kwargs):
 
         return Document(self.client, create_json)
 
-    def _collect_files(self, path, extension):
-        """Find the paths to all pdfs under a directory"""
+    def _collect_files(self, path):
```

Comment:
I would not remove the ability to specify the file extension.  I think a common use case would still be to upload all PDFs in a directory while skipping text and image files.  I think you should be able to either specify an extension or list of extensions you want to upload, or have a way to specify upload anything DocumentCloud can handle.